code4lib/2022.code4lib.org
# speakers - better twitter link for screenreaders

Tell readers it's a twitter link while not repeating the username twice. Twitter usernames tend to be unreadable and simply announcing something like "link: @phette23" is not helpful. Closes #48

To test:
- apply PR
- use a screenreader (e.g. Voiceover on macOS, System Preferences > Accessibility > Voiceover. If your system doesn't have a screenreader and you don't want to install one, there are Chrome extensions too)
- visit past keynotes page /speakers/past-keynotes
- expand the bio of someone with a Twitter handle (e.g. Mega Subramaniam)
- listen to the link being read before the bio to see if it makes sense
